File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/MonsterGroupConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class MonsterGroupConfigModel extends ConfigModel {
	public int group_gid;//怪组
	public String descs;//描述
	public ArrInt monsters;//卡牌名字
	public ArrInt monster_positions;//站位
	public ArrInt monster_levels;//等级
	public ArrInt monster_skill_lvs;//技能等级
	public ArrInt monster_skill_steps;//技能阶段
	public ArrInt heroes;//主角
	public ArrInt hero_positions;//站位
	public ArrInt hero_skill_lvs;//主角技能等级
	public ArrInt hero_skill_steps;//主角技能阶段
}

File: mobi_client/mobi_client/mobi_hotfix/mobi_hotfix/HFScripts/HFConfig/ConfigModel/MonsterGroupConfigModel.cs
public class MonsterGroupConfigModel : ConfigSystem.ConfigModel {
    public int group_gid;//怪组
    public string descs;//描述
    public System.Collections.Generic.List<int> monsters;//卡牌名字
    public System.Collections.Generic.List<int> monster_positions;//站位
    public System.Collections.Generic.List<int> monster_levels;//等级
    public System.Collections.Generic.List<int> monster_skill_lvs;//技能等级
    public System.Collections.Generic.List<int> monster_skill_steps;//技能阶段
    public System.Collections.Generic.List<int> heroes;//主角
    public System.Collections.Generic.List<int> hero_positions;//站位
    public System.Collections.Generic.List<int> hero_skill_lvs;//主角技能等级
    public System.Collections.Generic.List<int> hero_skill_steps;//主角技能阶段
}